package.json

```diff
@@ -46,8 +46,5 @@
     "through2": "2.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0"
-  },
-  "dependencies": {
-    "vsts-task-lib": "^0.9.20"
```

Comment:
i dont think there's a reason to have this. especially after the confusion i ran into with the tests, i'd rather remove it. it's not in master.